editor.html

```diff
@@ -74,40 +74,48 @@
             UPY_VERSION = "1.0.1";
         </script>
         <script id="load-template" type="x-tmpl-mustache">
-            <h2><i class="fa fa-upload"></i> <strong>{{ load-title }}</strong></h2>
+            <h2 style="margin-top: 0;"><i class="fa fa-file"></i> <strong>{{ load-title }}</strong></h2>
```

Comment:
Any reason to style like this only this modal and not the others?
If we can we should move this to the css file as well.